### PR TITLE
Prophet + WRMSSE

### PR DESCRIPTION
**Pull request by nnistelrooij**
The first baseline was implemented: `fbprophet` from Facebook. It received a score of 0.86, which was about the same effectiveness as an average baseline.

So another method was devised of which the loss function has now been implemented.
Pretty complicated stuff in the private functions, but the interface is still just `loss.backward()`, so the same as for the previous part of the course.

### CODE REVIEW

File: nn.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    prices = pd.read_csv(path + 'sell_prices.csv')
+    sales = pd.read_csv(path + 'sales_train_validation.csv')
+
+    device = torch.device('cpu')
+    time = datetime.now()
+    criterion = WRMSSE(device, calendar, prices, sales)
+    print('Time to initialize loss: ', datetime.now() - time)
+
+    input = torch.rand(30490, 5, device=device)
+    target = torch.rand(30490, 5)
```
Comment (rusane) [inline nn.py]: Does `5` represent the length of each input timeseries here (i.e., 5 days)?

**Comment by rusane** [comment]
Nice work, you can merge it in my opinion.

File: nn.py (hunk 1 above)
Comment (nnistelrooij) [reply nn.py]: Does the new commit answer your question?

Comment (rusane) [reply nn.py]: Yes, I was actually a bit confused about the `input` variable name as it is actually the supposed output of the model (predicted sales/forecast).

Comment (nnistelrooij) [reply nn.py]: I usually take over the names of the parameters that are used in the same method in the super class. I do agree that it's confusing, but it does fit better within the PyTorch framework.

I will merge this PR.
